modules/tinymce/src/core/test/ts/browser/annotate/AnnotationChangedTest.ts

```diff
@@ -295,14 +295,14 @@ describe('browser.tinymce.core.annotate.AnnotationChangedTest', () => {
     annotate(editor, 'beta', 'id-three', { something: 'comment-3' });
 
     // Content should not include the view only active state attributes
-    TinyAssertions.assertContent(editor, [
-      '<p>',
-      '<span class="mce-annotation" data-mce-annotation-uid="id-one" data-mce-annotation="alpha" data-test-anything="comment-1">one ',
-      '<span class="mce-annotation" data-mce-annotation-uid="id-two" data-mce-annotation="beta" data-test-something="comment-2">two ',
-      '<span class="mce-annotation" data-mce-annotation-uid="id-three" data-mce-annotation="beta" data-test-something="comment-3">three</span></span></span>',
+    assertHtmlContent(editor, [
```

Comment:
Why did this need to be changed the output should be deterministic when you call editor.getContent()?

Reply:
This doesn't look like a content change, to me, it's just swapping from array `join` to string append

Reply:
It's switching assert function as well not just the content.